features/rni.feature

```diff
@@ -0,0 +1,202 @@
+# language: nl
+
+Functionaliteit: RNI-deelnemer voor geleverde gegevens
+
+
+  Rule: de RNI-deelnemer gegevens worden geleverd voor elke categorie waarvan ten minste één gegeven gevraagd is
+    - RNI-deelnemer gegevens worden ook geleverd wanneer hier niet naar gevraagd is met de fields parameter
+    - categorie wordt gevuld met de naam van categorie waarop de RNI-deelnemer gegevens heeft aangeleverd
+      | categorie | naam           |
+      | 01        | Persoon        |
+      | 04        | Nationaliteit  |
+      | 06        | Overlijden     |
+      | 07        | Inschrijving   |
+      | 08        | Verblijfplaats |
+
+
+    Achtergrond:
+      Gegeven landelijke tabel "RNI-deelnemerstabel" heeft de volgende waarden
+      | code | omschrijving                                      |
+      | 101  | Belastingdienst (inzake heffingen en toeslagen)   |
+      | 201  | Sociale Verzekeringsbank (inzake AOW, Anw en AKW) |
+
+
+    Scenario: RNI-deelnemer op geleverd gegeven
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                            | waarde                                      |
+      | burgerservicenummer             | 000000012                                   |
+      | RNI-deelnemer (01.88.10)        | 101                                         |
+      | Omschrijving verdrag (01.88.20) | Belastingverdrag tussen België en Nederland |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
```

Comment:
Deze rij kan weggelaten worden, heeft in dit scenario geen functie.

Reply:
aangepast door ook geslachtsnaam toe te voegen aan fields